homeassistant/components/discord/notify.py

```diff
@@ -62,7 +62,7 @@ async def async_send_message(self, message, **kwargs):
         embed = None
         if ATTR_EMBED in data:
             embedding = data[ATTR_EMBED]
-            fields = embedding.get(ATTR_EMBED_FIELDS)
+            fields = embedding.get(ATTR_EMBED_FIELDS) or {}
```

Comment:
Fields should be a list of dicts.
```suggestion
            fields = embedding.get(ATTR_EMBED_FIELDS) or []
```